src/lib/comms/sol-coap.c

```diff
@@ -1438,6 +1457,19 @@ sol_coap_server_new(int port)
 }
 
 SOL_API int
+sol_coap_packet_get_buf(struct sol_coap_packet *pkt, uint8_t **buf, uint16_t *len)
```

Comment:
OK, but for me those comments were pretty much not conclusive.
What approach should be used? Chiossi's suggestion?